packages/terra-graphs-docs/src/terra-dev-site/doc/example/Graphs/BubbleGraph/ColorBasedGraph.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import BubbleSingleDataset from '@cerner/terra-graphs/lib/components/Bubble/BubbleSingleDataset';
+import utils from '@cerner/carbon-graphs/lib/js/helpers/utils';
+import '@cerner/terra-graphs/lib//components/Graph.module.scss';
+import '@cerner/terra-graphs/lib//components/Dev.module.scss';
+import getSimpleAxisData from '@cerner/terra-graphs-docs/src/example-datasets/graphConfigObjects/Bubble/simpleAxisData';
+import bubbleDataColorBased from '@cerner/terra-graphs-docs/src/example-datasets/dataObjects/Bubble/bubbleDataColorBased.js';
+
+/*
+Please refer to the documentation below to see graphConfig and data objects.
+*/
+const graphConfig = utils.deepClone(getSimpleAxisData('#colorBasedBubbleGraph'));
+const dataset = utils.deepClone(bubbleDataColorBased);
+
+export default () => (
+  <React.Fragment>
```

Comment:
Not a big deal, but I just want to point out that React has a shorthand syntax for fragments
```suggestion
  <>
  </>
```